pos_pricelist/static/src/js/models.js

```diff
@@ -116,16 +130,23 @@ function pos_pricelist_models(instance, module) {
     /**
      * Extend the Order line
      */
+    var OrderlineParent = module.Orderline;
     module.Orderline = module.Orderline.extend({
+        /**
+         * @param attr
+         * @param options
+         */
         initialize: function (attr, options) {
-            this._super('initialize', attr, options);
+            OrderlineParent.prototype.initialize.apply(this, arguments);
             this.manuel_price = false;
```

Comment:
Is this a typo-error? Manuel is my  son's name :smile:  Should it be manu**a**l_price?